awx/main/access.py

```diff
@@ -2949,6 +2950,22 @@ def filtered_queryset(self):
         return self.model.objects.filter(workflowjobtemplatenodes__workflow_job_template__in=WorkflowJobTemplate.accessible_pk_qs(self.user, 'read_role'))
 
 
+class PeersAccess(BaseAccess):
```

Comment:
This should presumably go away along with the peers endpoint

Reply:
Right now I'm not seeing any views which are not read-only.

Reply:
dropping this